### DISCUSSION TITLE
Editor lags if Osu!lazer left minimized in editor mode for some time

### DISCUSSION THREAD

**Discussion by maxmine2** [ppy/osu#28758] [Q&A] [answered]
On certain occasions, osu!lazer starts to lag if left in editor mode and window was minimized.

### This happens when I return to Osu! editor:
![image](https://github.com/ppy/osu/assets/44315846/a6e4be67-5250-48e2-93cb-56181126245e)

Besides, the zone about ~0.5 seconds in both directions from the point at which the windows was minimized and left for certain period of time is laggy with FPS falling up to around 12-14. However, if test it in normal gameplay, no lags appear.

Issue can be fixed with reloading osu!editor

Showcase: https://youtu.be/7RPTh6yrxS4

Edit: forgot to add logs: [logs_2.zip](https://github.com/user-attachments/files/16118913/logs_2.zip). There are two sessions, first one crashed during some manipulations in laggy area.

Lazer version: 2024.625.2

**Comment by jhey29** [comment]
It doesn't matter whether the editor is minimized. Having the editor open builds up lag. Unsure whether I should make this another post. According to my tiny amount of testing, the lag is especially bad if there are a lot of sliders on screen.

![osu_2024-07-07_14-56-04](https://github.com/ppy/osu/assets/110559474/77f9a320-33a3-4b3c-8dcb-cc19048449f9)

![osu_2024-07-07_14-56-17](https://github.com/ppy/osu/assets/110559474/3171d6a8-643c-4395-88b2-28cb44ccfbf6)

**Reply by maxmine2** [reply]
You have a good point, I guess. In my case, if I remember correctly, editor wasn't even set at any object, rather at the blank space after one of them, and still this bug presented itself, although I cannot confirm validity of this fact.

**Reply by maxmine2** [reply]
Besides, while I was searching for duplicate discussions before submitting this report, I have found #23293. I think that lazer editor is still not optimized perfectly, and that's why such issues occur. I don't think optimization and other lags connected to it would be a priority, maybe even until the lazer itself will be as complete as developer want it to be. Moreover, this lag is easily fixed with just relaunching editor, which takes seconds.

**Reply by peppy** [reply, marked as answer]
This will likely be fixed in the next release.

**Reply by maxmine2** [reply]
Thanks =)
